### PR TITLE
Websocket reconnects

### PR DESCRIPTION
**Pull request by pavel-kokolemin**

### CODE REVIEW

File: BlocksettleNetworkingLib/PublisherServerConnection.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+   void OnDataFromClient(const std::string& clientId, const std::string& data) override
+   {
+   }
+
+   void OnClientConnected(const std::string &clientId, const Details &details) override
+   {
+      std::lock_guard<std::mutex> lock(owner_->mutex_);
+      owner_->conn_->SendDataToClient(clientId, owner_->welcomeMsg_);
+      owner_->conn_->SendDataToClient(clientId, owner_->latestDataMsg_);
```
Comment (Ation) [inline BlocksettleNetworkingLib/PublisherServerConnection.cpp:26]: Why do we need to send latest message? Welcome message exists exactly for that

File: BlocksettleNetworkingLib/ZmqServerConnection.cpp [hunk 2]
```diff
@@ -230,24 +164,21 @@ void ZmqServerConnection::listenFunction()
...
-   poll_items[ZmqServerConnection::MonitorSocketIndex].events = ZMQ_POLLIN;
-
    logger_->debug("[{}] poll thread started for {}", __func__, connectionName_);
 
    int errorCount = 0;
 
    while (true) {
       int periodMs = std::chrono::duration_cast<std::chrono::milliseconds>(kHearthbeatCheckPeriod).count();
-      int result = zmq_poll(poll_items, 3, periodMs);
+      int result = zmq_poll(poll_items, sizeof(poll_items) / sizeof(poll_items[0]), periodMs);
```
Comment (Ation) [inline BlocksettleNetworkingLib/ZmqServerConnection.cpp:181]: please use constexpr. This is compile time variable

File: BlocksettleNetworkingLib/PublisherServerConnection.cpp (hunk 1 above)
Comment (pavel-kokolemin) [reply BlocksettleNetworkingLib/PublisherServerConnection.cpp]: Right, fixed, thanks

File: BlocksettleNetworkingLib/ZmqServerConnection.cpp (hunk 2 above)
Comment (pavel-kokolemin) [reply BlocksettleNetworkingLib/ZmqServerConnection.cpp]: Sure, should be fixed now. Trough I'm 99% sure this is already optimized by compiler.

**Pull request merged**